.pending/breaking/rest/_Remove-version-and-

```diff
@@ -0,0 +1,2 @@
+Remove /version and /node_version endpoints in favor of refactoring /node_info to also
```

Comment:
```suggestion
#4837 Remove /version and /node_version endpoints in favor of refactoring /node_info to also
```

Reply:
This is gonna automatically convert it to an issue link. When there is no issue I usually manually add the link. I'll do that.